hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -256,7 +256,9 @@ private synchronized void updateDefaultServers() {
     newGroupMap.put(RSGroupInfo.DEFAULT_GROUP, newDefaultGroupInfo);
     // do not need to persist, as we do not persist default group.
     resetRSGroupMap(newGroupMap);
-    LOG.info("Updated default servers, {} servers", newDefaultGroupInfo.getServers().size());
+    LOG.info("Updated default servers, now {} servers online: {}",
+      newDefaultGroupInfo.getServers().size(),
+      newDefaultGroupInfo.getServers().stream().map(Address::toString).collect(Collectors.toSet()));
```

Comment:
I think the message will be too long if we log the servers? This is a log in info level, not debug.